registry_inspector/tests/test_inspector.py

```diff
@@ -0,0 +1,41 @@
+from registry_inspector.inspector import RegistryInspector
+import responses
+
+
+def layer_callback(sha256, content_length):
+        def cb(request):
+            headers = {
+                    "Docker-Content-Digest": "sha256:%s" % (sha256,),
+                    "Etag": "sha256: %s" % (sha256,),
+                    "Content-Length": content_length,
+            }
+            return 200, headers, ""
+        return cb
+
+
+@responses.activate
+def test_get_layer_dict():
+
+    registry = 'http://localhost:5000'
+    name = 'ubuntu'
+    digest = {('a3ed95caeb02ffe68cdd9fd84406680ae93d633'
+              'cb16422d00e8a7c22955b46d4'): '32', ('10fb34ebccea88897d4b5701'
+              '20719b23fcbf3e556abdb79fdba6b2e0e4bad9ab'): '680'}
```

Comment:
You're using these hashes a lot, and trying to fit them into 80 character lines is ugly. Why not make them constants at the top of the module and then reference those in the tests?
```
BLOB1 = 'a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4'
BLOB2 = '10fb34ebccea88897d4b570120719b23fcbf3e556abdb79fdba6b2e0e4bad9ab'
...
    digest = {BLOB1: '32', BLOB2: '680'}
...
```